Files gail.orig/.libs/libgail.dll and gail/.libs/libgail.dll differ
Files gail.orig/.libs/libgail.dll.a and gail/.libs/libgail.dll.a differ
diff -Nru gail.orig/.libs/libgail.la gail/.libs/libgail.la
--- gail.orig/.libs/libgail.la	1970-01-01 09:00:00 +0900
+++ gail/.libs/libgail.la	2011-03-02 20:51:49 +0900
@@ -0,0 +1,41 @@
+# libgail.la - a libtool library file
+# Generated by libtool (GNU libtool) 2.4
+#
+# Please DO NOT delete this file!
+# It is necessary for linking the library.
+
+# The name that we can dlopen(3).
+dlname='libgail.dll'
+
+# Names of this library.
+library_names='libgail.dll.a'
+
+# The name of the static archive.
+old_library=''
+
+# Linker flags that can not go in dependency_libs.
+inherited_linker_flags=''
+
+# Libraries that this one depends upon.
+dependency_libs=' -R/x86_64-w64-mingw32/local/lib  -L/x86_64-w64-mingw32/local/lib -L/x86_64-w64-mingw32/glib/lib -LC:/x86_64-w64-mingw32/local/lib -L/c/bb/vista64-mingw32/mingw-x86-x86_64/build/build/gcc/obj/x86_64-w64-mingw32/libstdc++-v3/src -L/c/bb/vista64-mingw32/mingw-x86-x86_64/build/build/gcc/obj/x86_64-w64-mingw32/libstdc++-v3/src/.libs -L/c/bb/vista64-mingw32/mingw-x86-x86_64/build/build/root/x86_64-w64-mingw32/lib -L/c/bb/vista64-mingw32/mingw-x86-x86_64/build/build/root/mingw/lib /x86_64-w64-mingw32/glib/src/gtk+-3.0.0/modules/other/gail/libgail-util/libgailutil-3.la /x86_64-w64-mingw32/glib/src/gtk+-3.0.0/gtk/libgtk-3.la /x86_64-w64-mingw32/glib/src/gtk+-3.0.0/gdk/libgdk-3.la /x86_64-w64-mingw32/glib/lib/libgdk_pixbuf-2.0.la /x86_64-w64-mingw32/glib/lib/libgio-2.0.la /x86_64-w64-mingw32/local/lib/libjpeg.la /x86_64-w64-mingw32/local/lib/libpng.la /x86_64-w64-mingw32/glib/lib/libpangocairo-1.0.la /x86_64-w64-mingw32/glib/lib/libpangoft2-1.0.la /c/bb/vista64-mingw32/mingw-x86-x86_64/build/build/root/x86_64-w64-mingw32/lib/../lib64/libstdc++.la /x86_64-w64-mingw32/glib/lib/libcairo-gobject.la /x86_64-w64-mingw32/glib/lib/libatk-1.0.la /x86_64-w64-mingw32/glib/lib/libpangowin32-1.0.la /x86_64-w64-mingw32/glib/lib/libpango-1.0.la /x86_64-w64-mingw32/glib/lib/libgobject-2.0.la /x86_64-w64-mingw32/glib/lib/libgthread-2.0.la /x86_64-w64-mingw32/glib/lib/libgmodule-2.0.la /x86_64-w64-mingw32/glib/lib/libcairo.la /x86_64-w64-mingw32/local/lib/libpixman-1.la /x86_64-w64-mingw32/local/lib/libfontconfig.la /x86_64-w64-mingw32/local/lib/libxml2.la /x86_64-w64-mingw32/local/lib/libfreetype.la /x86_64-w64-mingw32/local/lib/libpng14.la /x86_64-w64-mingw32/local/lib/libz.la /x86_64-w64-mingw32/glib/lib/libglib-2.0.la /x86_64-w64-mingw32/local/lib/libintl.la /x86_64-w64-mingw32/local/lib/libiconv.la'
+
+# Names of additional weak libraries provided by this library
+weak_library_names=''
+
+# Version information for libgail.
+current=0
+age=0
+revision=0
+
+# Is this an already installed library?
+installed=no
+
+# Should we warn about portability when linking against -modules?
+shouldnotlink=yes
+
+# Files to dlopen/dlpreopen
+dlopen=''
+dlpreopen=''
+
+# Directory that this library needs to be installed in:
+libdir='/x86_64-w64-mingw32/glib/lib/gtk-3.0/modules'
diff -Nru gail.orig/.libs/libgail.lai gail/.libs/libgail.lai
--- gail.orig/.libs/libgail.lai	1970-01-01 09:00:00 +0900
+++ gail/.libs/libgail.lai	2011-03-02 20:51:49 +0900
@@ -0,0 +1,41 @@
+# libgail.la - a libtool library file
+# Generated by libtool (GNU libtool) 2.4
+#
+# Please DO NOT delete this file!
+# It is necessary for linking the library.
+
+# The name that we can dlopen(3).
+dlname='libgail.dll'
+
+# Names of this library.
+library_names='libgail.dll.a'
+
+# The name of the static archive.
+old_library=''
+
+# Linker flags that can not go in dependency_libs.
+inherited_linker_flags=''
+
+# Libraries that this one depends upon.
+dependency_libs=' -R/x86_64-w64-mingw32/local/lib -L/x86_64-w64-mingw32/local/lib -L/x86_64-w64-mingw32/glib/lib -L=C:/x86_64-w64-mingw32/local/lib -L/c/bb/vista64-mingw32/mingw-x86-x86_64/build/build/gcc/obj/x86_64-w64-mingw32/libstdc++-v3/src -L/c/bb/vista64-mingw32/mingw-x86-x86_64/build/build/gcc/obj/x86_64-w64-mingw32/libstdc++-v3/src/.libs -L/c/bb/vista64-mingw32/mingw-x86-x86_64/build/build/root/x86_64-w64-mingw32/lib -L/c/bb/vista64-mingw32/mingw-x86-x86_64/build/build/root/mingw/lib /x86_64-w64-mingw32/glib/lib/libgailutil-3.la /x86_64-w64-mingw32/glib/lib/libgtk-3.la /x86_64-w64-mingw32/glib/lib/libgdk-3.la /x86_64-w64-mingw32/glib/lib/libgdk_pixbuf-2.0.la /x86_64-w64-mingw32/glib/lib/libgio-2.0.la /x86_64-w64-mingw32/local/lib/libjpeg.la /x86_64-w64-mingw32/local/lib/libpng.la /x86_64-w64-mingw32/glib/lib/libpangocairo-1.0.la /x86_64-w64-mingw32/glib/lib/libpangoft2-1.0.la /c/bb/vista64-mingw32/mingw-x86-x86_64/build/build/root/x86_64-w64-mingw32/lib/../lib64/libstdc++.la /x86_64-w64-mingw32/glib/lib/libcairo-gobject.la /x86_64-w64-mingw32/glib/lib/libatk-1.0.la /x86_64-w64-mingw32/glib/lib/libpangowin32-1.0.la /x86_64-w64-mingw32/glib/lib/libpango-1.0.la /x86_64-w64-mingw32/glib/lib/libgobject-2.0.la /x86_64-w64-mingw32/glib/lib/libgthread-2.0.la /x86_64-w64-mingw32/glib/lib/libgmodule-2.0.la /x86_64-w64-mingw32/glib/lib/libcairo.la /x86_64-w64-mingw32/local/lib/libpixman-1.la /x86_64-w64-mingw32/local/lib/libfontconfig.la /x86_64-w64-mingw32/local/lib/libxml2.la /x86_64-w64-mingw32/local/lib/libfreetype.la /x86_64-w64-mingw32/local/lib/libpng14.la //x86_64-w64-mingw32/local/lib/libz.la /x86_64-w64-mingw32/glib/lib/libglib-2.0.la /x86_64-w64-mingw32/local/lib/libintl.la /x86_64-w64-mingw32/local/lib/libiconv.la'
+
+# Names of additional weak libraries provided by this library
+weak_library_names=''
+
+# Version information for libgail.
+current=0
+age=0
+revision=0
+
+# Is this an already installed library?
+installed=yes
+
+# Should we warn about portability when linking against -modules?
+shouldnotlink=yes
+
+# Files to dlopen/dlpreopen
+dlopen=''
+dlpreopen=''
+
+# Directory that this library needs to be installed in:
+libdir='/x86_64-w64-mingw32/glib/lib/gtk-3.0/modules'
Files gail.orig/.libs/libgail_la-gail.o and gail/.libs/libgail_la-gail.o differ
Files gail.orig/.libs/libgail_la-gailtoplevel.o and gail/.libs/libgail_la-gailtoplevel.o differ
Files gail.orig/.libs/libgail_la-gailwindow.o and gail/.libs/libgail_la-gailwindow.o differ
diff -Nru gail.orig/gail.c gail/gail.c
--- gail.orig/gail.c	2011-01-31 11:24:51 +0900
+++ gail/gail.c	2011-03-02 20:48:46 +0900
@@ -307,9 +307,11 @@
    * then ignore the focus notification as the embedded
    * plug will report a focus notification.
    */
+#ifdef GDK_WINDOWING_X11
   if (GTK_IS_SOCKET (widget) &&
       gtk_socket_get_plug_window (GTK_SOCKET (widget)) != NULL)
     return TRUE;
+#endif
   /*
    * The widget may not yet be visible on the screen so we wait until it is.
    */
diff -Nru gail.orig/gailtoplevel.c gail/gailtoplevel.c
--- gail.orig/gailtoplevel.c	2011-01-31 11:24:51 +0900
+++ gail/gailtoplevel.c	2011-03-02 20:50:30 +0900
@@ -94,8 +94,11 @@
       if (!window || 
           !gtk_widget_get_visible (widget) ||
           is_attached_menu_window (widget) ||
-          gtk_widget_get_parent (GTK_WIDGET (window)) ||
-          GTK_IS_PLUG (window))
+          gtk_widget_get_parent (GTK_WIDGET (window))
+#ifdef GDK_WINDOWING_X11
+          || GTK_IS_PLUG (window)
+#endif
+          )
         {
           GList *temp_l  = l->next;
 
@@ -215,8 +218,11 @@
   widget = GTK_WIDGET (object);
   if (gtk_widget_get_parent (widget) ||
       is_attached_menu_window (widget) ||
-      is_combo_window (widget) ||
-      GTK_IS_PLUG (widget))
+      is_combo_window (widget)
+#ifdef GDK_WINDOWING_X11
+      || GTK_IS_PLUG (widget)
+#endif
+      )
     return TRUE;
 
   child = gtk_widget_get_accessible (widget);
diff -Nru gail.orig/gailwindow.c gail/gailwindow.c
--- gail.orig/gailwindow.c	2011-01-31 11:24:51 +0900
+++ gail/gailwindow.c	2011-03-02 20:51:20 +0900
@@ -226,8 +226,10 @@
 
       if (!g_strcmp0 (name, "gtk-tooltip"))
         obj->role = ATK_ROLE_TOOL_TIP;
+#ifdef GDK_WINDOWING_X11
       else if (GTK_IS_PLUG (widget))
         obj->role = ATK_ROLE_PANEL;
+#endif
       else if (gtk_window_get_window_type (GTK_WINDOW (widget)) == GTK_WINDOW_POPUP)
         obj->role = ATK_ROLE_WINDOW;
       else
diff -Nru gail.orig/libgail.la gail/libgail.la
--- gail.orig/libgail.la	1970-01-01 09:00:00 +0900
+++ gail/libgail.la	2011-03-02 20:51:49 +0900
@@ -0,0 +1,41 @@
+# libgail.la - a libtool library file
+# Generated by libtool (GNU libtool) 2.4
+#
+# Please DO NOT delete this file!
+# It is necessary for linking the library.
+
+# The name that we can dlopen(3).
+dlname='libgail.dll'
+
+# Names of this library.
+library_names='libgail.dll.a'
+
+# The name of the static archive.
+old_library=''
+
+# Linker flags that can not go in dependency_libs.
+inherited_linker_flags=''
+
+# Libraries that this one depends upon.
+dependency_libs=' -R/x86_64-w64-mingw32/local/lib  -L/x86_64-w64-mingw32/local/lib -L/x86_64-w64-mingw32/glib/lib -LC:/x86_64-w64-mingw32/local/lib -L/c/bb/vista64-mingw32/mingw-x86-x86_64/build/build/gcc/obj/x86_64-w64-mingw32/libstdc++-v3/src -L/c/bb/vista64-mingw32/mingw-x86-x86_64/build/build/gcc/obj/x86_64-w64-mingw32/libstdc++-v3/src/.libs -L/c/bb/vista64-mingw32/mingw-x86-x86_64/build/build/root/x86_64-w64-mingw32/lib -L/c/bb/vista64-mingw32/mingw-x86-x86_64/build/build/root/mingw/lib /x86_64-w64-mingw32/glib/src/gtk+-3.0.0/modules/other/gail/libgail-util/libgailutil-3.la /x86_64-w64-mingw32/glib/src/gtk+-3.0.0/gtk/libgtk-3.la /x86_64-w64-mingw32/glib/src/gtk+-3.0.0/gdk/libgdk-3.la /x86_64-w64-mingw32/glib/lib/libgdk_pixbuf-2.0.la /x86_64-w64-mingw32/glib/lib/libgio-2.0.la /x86_64-w64-mingw32/local/lib/libjpeg.la /x86_64-w64-mingw32/local/lib/libpng.la /x86_64-w64-mingw32/glib/lib/libpangocairo-1.0.la /x86_64-w64-mingw32/glib/lib/libpangoft2-1.0.la /c/bb/vista64-mingw32/mingw-x86-x86_64/build/build/root/x86_64-w64-mingw32/lib/../lib64/libstdc++.la /x86_64-w64-mingw32/glib/lib/libcairo-gobject.la /x86_64-w64-mingw32/glib/lib/libatk-1.0.la /x86_64-w64-mingw32/glib/lib/libpangowin32-1.0.la /x86_64-w64-mingw32/glib/lib/libpango-1.0.la /x86_64-w64-mingw32/glib/lib/libgobject-2.0.la /x86_64-w64-mingw32/glib/lib/libgthread-2.0.la /x86_64-w64-mingw32/glib/lib/libgmodule-2.0.la /x86_64-w64-mingw32/glib/lib/libcairo.la /x86_64-w64-mingw32/local/lib/libpixman-1.la /x86_64-w64-mingw32/local/lib/libfontconfig.la /x86_64-w64-mingw32/local/lib/libxml2.la /x86_64-w64-mingw32/local/lib/libfreetype.la /x86_64-w64-mingw32/local/lib/libpng14.la /x86_64-w64-mingw32/local/lib/libz.la /x86_64-w64-mingw32/glib/lib/libglib-2.0.la /x86_64-w64-mingw32/local/lib/libintl.la /x86_64-w64-mingw32/local/lib/libiconv.la'
+
+# Names of additional weak libraries provided by this library
+weak_library_names=''
+
+# Version information for libgail.
+current=0
+age=0
+revision=0
+
+# Is this an already installed library?
+installed=no
+
+# Should we warn about portability when linking against -modules?
+shouldnotlink=yes
+
+# Files to dlopen/dlpreopen
+dlopen=''
+dlpreopen=''
+
+# Directory that this library needs to be installed in:
+libdir='/x86_64-w64-mingw32/glib/lib/gtk-3.0/modules'
